packages/models/organizations.coffee

```diff
@@ -7,8 +7,13 @@ Organization = Astro.Class
     name: 'string'
     description: 'string'
     createdById: 'string'
-
   methods:
+    addMember: (userId) ->
+      UserProfiles.update({userId: userId}, {
+        $addToSet: {memberOfOrgs: @_id}
+      })
```

Comment:
If we use the the allow/deny feature to prevent all client-side modifications of the collection, this method would in effect be a server-only method.